data.sql

```diff
@@ -4,3 +4,8 @@ INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('A
 INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Gabumon','2018/11/15',1,FALSE,8);
 INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Pikachu','2021/07/1',2,FALSE,15.04);
 INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Devimon','2017/05/12',5,TRUE,11.0);
+--Other insertions
+INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Squirtle','1993-04-2',3,FALSE,-12);
+INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Angemon','2005-06-12',1,TRUE,-45);
+INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Boarmon','2005-06-07',7,TRUE,20.4);
+INSERT INTO animals(name,date_of_birth,escape_attempt,neutered,weight) VALUES('Ditto','2022-05-14',4,TRUE,22);
```

Comment:
- [ ] Please kindly include commands to insert the following
  - Chamander
  - Plantmon
  - Blossom